CONTRIBUTING.md

```diff
@@ -0,0 +1,127 @@
+# Contributing to Quizard (Frontend)
+
+<!-- Add a thank you message for new/existing contributors -->
+
+To get an overview of the project, look at the [README.md](README.md) file.
+
+This guide will cover the contribution workflow from opening an issue, creating a PR, reviewing, and merging the PR.
+
+Use the table of contents icon <img src="public/assets/table-of-contents.png" width="24" height="24" /> on the top left corner of this document to get to a specific section quickly.
+
+# Ground Rules
+
+Read our [Code of Conduct](CODE_OF_CONDUCT.md) to understand how you can help keep our community approachable and respectable.
+
+### A quick summary
```

Comment:
Is this necessary? if it is, then include it.

Reply:
Yeah it is. David will work on it as well

Basically everything in the comments is what he's focusing on